proposals/src/mappings/mappingHandlers.ts

```diff
@@ -134,50 +150,73 @@ export async function handlePropose(
   let proposalNumber: number;
   try {
     proposalNumber = Number(
-      findAttribute(JSON.parse(cosmosMessage.tx.tx.log), "wasm", "proposal_id")
-        .value
+      findAttribute(JSON.parse(log), "wasm", "proposal_id").value
```

Comment:
sdk / cosmwasm stuff: if there is more than one message in the transaction you need to find the wasm events corresponding to your transaction, otherwise we risk putting incorrect info here if there are multiple messages in one TX. you should be able to reuse the code here for this: https://github.com/DA0-DA0/subquery-indexers/blob/2ed953b725403fd2f0919006c39835cee5c35f96/wasmswap-indexer/src/mappings/mappingHandlers.ts#L38-L57

Reply:
ah crap did not realize that since this is a message handler, so cool it gives us the index! will fix this thanks